tests/ethercat/test_pdo.py

```diff
@@ -241,6 +241,54 @@ def test_servo_add_maps(connect_to_slave, create_pdo_map):
     assert int.to_bytes(0x1600, 2, "little") == value[2:4]
 
 
+@pytest.mark.ethercat
+def test_modifying_pdos_prevented_if_servo_in_operational_state(connect_to_slave):
+    _, net = connect_to_slave
+
+    operation_mode_uid = "DRV_OP_CMD"
+    rpdo_registers = [operation_mode_uid]
+    operation_mode_display_uid = "DRV_OP_VALUE"
+    tpdo_registers = [operation_mode_display_uid]
+    default_operation_mode = 1
+    current_operation_mode = {}
+    new_operation_mode = {}
+    for index, servo in enumerate(net.servos):
```

Comment:
connect_to_slave yields exactly one servo and one net. https://github.com/ingeniamc/ingenialink-python/blob/decac4aef91665785aeea7853b1efd6462026fd6/tests/conftest.py#L104

Isn't iterating through this making the test more complex?